addon/lib/experiment-hacks.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * This Source Code is subject to the terms of the Mozilla Public License
+ * version 2.0 (the 'License'). You can obtain a copy of the License at
+ * http://mozilla.org/MPL/2.0/.
+ */
+
+const PrefsService = require('sdk/preferences/service');
+const store = require('sdk/simple-storage').storage;
+
+const BLOK_ID = 'blok@mozilla.org';
+const TP_PREFS = [
+  'privacy.trackingprotection.enabled',
+  'privacy.trackingprotection.pbmode.enabled',
+  'services.sync.prefs.sync.privacy.trackingprotection.enabled',
+  'services.sync.prefs.sync.privacy.trackingprotection.pbmode.enabled'
```

Comment:
Sync prefs weren't in the original bug, but we probably want to turn off syncing of these prefs too. @clouserw @johngruen ?